examples/monolithic_build/mlkem_native_all.c

```diff
@@ -74,11 +69,6 @@
 #undef _16XMONTSQLO
 #endif
 
-/* mlkem/poly.h:715 */
-#if defined(poly_add)
```

Comment:
How are the undefs ordered? There are a lot of changes to this file in this commit. Many more than necessary. Can we order the undefs by file?

Reply:
Yes, good point. Using file+line_no ordering now.